docs/specifying-return-values.md

```diff
@@ -63,8 +63,8 @@ than 4 parameters, the convenience methods won't work. Use the variant
 that takes an `IFakeObjectCall` instead:
 
 ```charp
-A.CallTo(objectCall => fakeShop.SomeCall(…)
-                               .ReturnsLazily(objectCall => calculateReturnFrom(objectCall));
+A.CallTo(() => fakeShop.SomeCall(…)
+                       .ReturnsLazily(objectCall => calculateReturnFrom(objectCall));

```

Comment:
Don't be sorry. The three owners missed the mistakes in the original docs, and then 2/3 owners also missed that missing paren. You've made things much better. Thanks!